changelog.d/13652.feature

```diff
@@ -0,0 +1 @@
+Improve aesthetics of HTML templates.
```

Comment:
```suggestion
Improve aesthetics of HTML templates. Note that these changes do not retroactively apply to templates which have been [customised](https://matrix-org.github.io/synapse/latest/templates.html#templates) by server admins.
```

Reply:
I agree, I don't think we need an upgrade note since no action is *required*.